docs/articles_en/learn_openvino/openvino_samples/get_started_demos.md

```diff
@@ -7,293 +7,330 @@
 
+.. tab-set::
 
-   Your screen will look similar to this after the download and show the paths of downloaded files:
+   .. tab-item:: Linux
+      :sync: linux
 
-   .. tab-set::
+      The officially supported Linux build environment is the following:
+
+      * Ubuntu 18.04 LTS 64-bit or Ubuntu 20.04 LTS 64-bit
+      * GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
+      * CMake version 3.10 or higher
+
+      .. tab-set::
+
+         .. tab-item:: Python
+            :sync: python
+   
+            Python samples do not require building. You can run the code samples in your development environment.
+   
+         .. tab-item:: C and C++
+            :sync: cpp
+   
+            To build the C or C++ sample applications for Linux, go to the ``<INSTALL_DIR>/samples/c`` or ``<INSTALL_DIR>/samples/cpp`` directory, respectively, and run the ``build_samples.sh`` script:
+      
+            .. code-block:: sh
+               
+               build_samples.sh
+      
+            Once the build is completed, you can find sample binaries in the following folders:
+      
+            * C samples: ``~/openvino_c_samples_build/<architecture>/Release``
+            * C++ samples: ``~/openvino_cpp_samples_build/<architecture>/Release`` where the <architecture> is the output of ``uname -m``, for example, ``intel64``, ``armhf``, or ``aarch64``.
+            
+            You can also build the sample applications manually:
+            
+            .. note::
+               
+               If you have installed the product as a root user, switch to root mode before you continue: ``sudo -i`` .
+      
+            1. Navigate to a directory that you have write access to and create a samples build directory. This example uses a directory named ``build``:
+            
+               .. code-block:: sh
+                  
+                  mkdir build
+               
+               .. note:: 
+               
+                  If you ran the Image Classification verification script during the installation, the C++ samples build directory is created in your home directory: ``~/openvino_cpp_samples_build/``
+               
+            2. Go to the created directory:
+               
+               .. code-block:: sh
+                  
+                  cd build
+               
+            3. Run CMake to generate the Make files for release or debug configuration. For example, for C++ samples:
+               
+               - For release configuration:
+            
+                 .. code-block:: sh
+                  
+                    cmake -DCMAKE_BUILD_TYPE=Release <INSTALL_DIR>/samples/cpp
+               
+               - For debug configuration:
+            
+                 .. code-block:: sh
+                    
+                    cmake -DCMAKE_BUILD_TYPE=Debug <INSTALL_DIR>/samples/cpp
```

Comment:
I think we can remove demonstration of building samples in Debug.
It's not related to samples, but rather to cmake features